layouts/partials/fragments/search.html

```diff
@@ -39,38 +39,32 @@ <h6>
         </div>
       </div>
     {{- end }}
-    <div class="row">
-      <form action="{{ "search" | absURL }}" class="w-100">
-        <div class="col-md-6 col-12">
-          <div class="form-group row">
-            <label for="search-query" class="col-2 pl-0 pt-2">Search:</label>
-            <input type="text" class="form-control col-10 col-md-8" id="search-query-{{ .Name }}" name="s" placeholder="Search for something...">
+    <div class="row w-100 justify-content-center">
+      <form action="{{ "search" | absURL }}" class="my-0">
+          <div class="input-group my-0">
+            <input type="text" class="form-control py-1" id="search-query-{{ .Name }}" name="s" placeholder="{{- i18n "navbar.search" | default "Search..." -}}">
+            <div class="input-group-append">
+              <button class="btn btn-primary"><i class="fa fa-search"></i></button>
+            </div>
           </div>
-        </div>
-      </form>
+        </form>
     </div>
     <div class="row">
       <div class="col px-0">
-        <h3>Matching pages</h3>
         <div id="search-results-{{ .Name }}" class="col px-0">
         </div>
       </div>
     </div>
   </div>
 </section>
-<!-- this template is sucked in by search.js and appended to the search-results div above. So editing here will adjust style -->
-<script id="search-result-template-{{ .Name }}" type="text/x-js-template">
-  <div id="summary-${key}">
-    <h4><a href="${link}">${title}</a></h4>
-    <p>${snippet}</p>
-    ${ isset tags }<p>Tags: ${tags}</p>${ end }
-    ${ isset categories }<p>Categories: ${categories}</p>${ end }
-  </div>
-</script>
+{{ partial "helpers/search-result-template.html" (dict "name" .Name "message_class" "text-center mt-4") }}
+{{ partial "helpers/search-result-template.html" . }}
```

Comment:
I assume calling it twice populates the page and the other populates the overlay right?

Reply:
OK I'm missing something. What's the reason for the second partial call with the whole site data?